src/cmd/api/Authentication.py

```diff
@@ -0,0 +1,35 @@
+import bcrypt
```

Comment:
this Authentication file should be in the pkg folder not In the cmd (executables) folder